pandas/core/series.py

```diff
@@ -926,6 +926,18 @@ def _repr_footer(self):
                                                len(self),
                                                str(self.dtype.name))
 
+    def _repr_html_(self, *args, **kwargs):
+        df = self.to_frame()
+        if self.name is None:
+            df.columns = ['']
+        return df._repr_html_(*args, **kwargs)
+
+    def to_html(self, *args, **kwargs):
```

Comment:
Please add the function to ``api.rst``, and better to use explicit keywords to render API doc.